src/app/views/communities/view/overview/overview.ts

```diff
@@ -104,11 +105,53 @@ export default class RouteCommunitiesViewOverview extends BaseRouteComponent {
 	emitRefresh() {}
 
 	get routeTitle() {
-		return this.community
-			? this.$gettextInterpolate(`%{ community } Community`, {
-					community: this.community.name,
-			  })
-			: null;
+		if (!this.community) {
+			return null;
+		}
+
+		let title = this.$gettextInterpolate(
+			`%{ name } Community - Fan Art, Videos, Guides, Polls and More`,
+			{
+				name: this.community.name,
+			}
+		);
+
+		if (this.channel === 'featured' || (this.sort !== 'hot' && this.sort !== 'new')) {
+			return title;
+		}
+
+		if (this.channel === 'all') {
+			switch (this.sort) {
+				case 'hot':
+					title = this.$gettext('Hot Posts');
```

Comment:
Hot posts, New posts

Reply:
what do you mean? when its hot literally start the title with "Hot posts, New posts"?

Reply:
He means no title case

Reply:
Oh. I think we should keep title case tho. any specific reason not to?